realm/realm-library/src/main/java/io/realm/RealmList.java

```diff
@@ -470,11 +477,13 @@ public E first() {
      * {@inheritDoc}
      */
     @Override
-    public E first(E defaultValue) {
+    @Nullable
```

Comment:
Not sure, but in parameters, overriding methods are required to have `@Nonnull` annotation if overridden method has `@Nonnull`.

So I guess each overriding methods should have `@Nullable` for return type.